generator/src/components/Button/index.svelte

```diff
@@ -0,0 +1,34 @@
+<script lang="ts">
+  export let onClick: () => void
+  export let icon: string | null = null
+  export let title: string
+</script>
+
+<button class="button" on:click|preventDefault={onClick}>
+  {#if icon}
+    <img class="icon" src={icon} alt="icon" />
```

Comment:
We could use `aria-hidden="true"` on the images like this. I mean if image doesn't supposed to be the source of information itself or has the corresponding description text - then we definitely should hide it from screen readers

Reply:
I don't think `alt="icon"` is a good answer here. How you should consider it is if someone was reading this out loud (like a screen reader would be), does reading the contents of the `alt` make it clearer? Saying "icon" is not helpful, the `title` is enough, so `alt="" aria-hidden="true"` is likely what we'd want here instead.